thumbor/config.py

```diff
@@ -131,6 +131,7 @@
 Config.define('ALLOW_OLD_URLS', True, 'Indicates if encrypted (old style) URLs should be allowed', 'Security')
 Config.define('ENABLE_ETAGS', True, 'Enables automatically generated etags', 'HTTP')
 Config.define('MAX_ID_LENGTH', 32, 'Set maximum id length for images when stored', 'Storage')
+Config.define('GC_INTERVAL', 60, 'Set garbage collection interval in seconds', 'Performance')
```

Comment:
Is it a breaking change? It doesn't change any image functionality and should reduce the probability of a memory leak taking their server down.

I think this issue is more likely to affect people than not. Meaning that it should probably be on by default. Otherwise users are left to first experience the bug, then google for it, find a github issue/PR and try to adjust thumbor config.